source/extensions/transport_sockets/tls/utility.cc

```diff
@@ -157,15 +157,19 @@ std::string Utility::generalNameAsString(const GENERAL_NAME* general_name) {
       sin.sin_port = 0;
       sin.sin_family = AF_INET;
       safeMemcpyUnsafeSrc(&sin.sin_addr, general_name->d.ip->data);
-      Network::Address::Ipv4Instance addr(&sin);
-      san = addr.ip()->addressAsString();
+      StatusOr<Network::Address::Ipv4Instance> error_or_addr =
+          Network::Address::InstanceFactory::createInstance<Network::Address::Ipv4Instance>(&sin);
+      ASSERT(error_or_addr.ok());
```

Comment:
RELEASE_ASSERT and don't swallow the error message.. Or *`OrDie`

Reply:
Makes sense.